recipe/meta.yaml

```diff
@@ -35,21 +39,54 @@ requirements:
     # need a CDT for libudev for cos6, proper package available for cos7 (below)
     - {{ cdt('libudev-devel') }}  # [linux and cdt_name=='cos6']
   host:
+    - python
     - libudev  # [linux and cdt_name!='cos6']
     - libusb
   run:
     - __cuda  # [cuda_compiler_version != "None"]
 
-test:
-  commands:
-    - test -f ${PREFIX}/include/librealsense2/rs.hpp  # [not win]
-    - test -f ${PREFIX}/lib/librealsense2.so  # [linux]
-    - test -f ${PREFIX}/lib/librealsense2.dylib  # [osx]
-    - test -f ${PREFIX}/lib/cmake/realsense2/realsense2Config.cmake  # [not win]
-    - if exist %PREFIX%\\Library\\include\\librealsense2\\rs.hpp (exit 0) else (exit 1)  # [win]
-    - if exist $PREFIX$\\Library\\lib\\realsense2.lib (exit 0) else (exit 1)  # [win]
-    - if exist $PREFIX$\\Library\\bin\\realsense2.dll (exit 0) else (exit 1)  # [win]
-    - if exist %PREFIX%\\Library\\lib\cmake\\realsense2\\realsense2Config.cmake (exit 0) else (exit 1)  # [win]
+
+outputs:
+  - name: librealsense
```

Comment:
I am probably missing something, but I do not get why for the  `librealsense` output, we specify again build and requirements options. See for example in https://github.com/conda-forge/opencv-feedstock/blob/891639f983996e8f7cae4595b53d7293cd31058b/recipe/meta.yaml#L201, for the `libopencv` output is empty because it is the main output of the recipe, and all the options are already specified as part of the top level build/requirements/test options.

Reply:
I see, I didn't realize the output config would inherit the root requirements but that does make sense; thought `libopencv` was a meta-package just for version pinning. Removed it from `librealsense`.